src/D3MHub.sol

```diff
@@ -331,6 +331,16 @@ contract D3MHub {
         emit Unwind(ilk, amount, fees);
     }
 
+    function _fix(bytes32 ilk, address _pool) internal returns (uint256 diff) {
```

Comment:
A note on the naming, if we're going with the change I think some daiwanese could be explored, consider something like `mend`, `give`, `neat`, `tidy`, `keel`.
I personally like the last one, `keel` as it seems like a [special one](https://en.wikipedia.org/wiki/Keel), it comes from the Latin `Carina`, is the origin of the term `careen`, to clean a keel and regarded by some scholars as the first word in the English language recorded in writing.

Reply:
Always up for daiwanese but don't forget it's an internal function.

Reply:
yeah, I shared for discussion in case we'd like to explore alternative naming.
If we'd like to stick with a 3 letters for an internal function, maybe `pop` could be an alternative.